### PR TITLE
Gitpod integration

### PR DESCRIPTION
**Pull request by jwpjrdev**
see #33

### CODE REVIEW

**Comment by jwpjrdev** [comment]
This could be sped up by a modification to aero.py that would only run QEMU rather than rebuilding and running. It creates the context by building aero but it has to rebuild and run when the workspace is reloaded/opened, which takes a little while.

File: aero.py [hunk 1]
```diff
@@ -638,7 +643,10 @@ def main():
 
     download_bundled()
 
-    if args.clean:
+    if args.only_run:
+        iso_path = os.path.join(BUILD_DIR, 'aero.iso')
```
Comment (Andy-Python-Programmer) [inline aero.py:647]: Here you should check if the iso file exists or not. If it doesn't, then it should build it first.

**Review by Andy-Python-Programmer — APPROVED** [review]
LGTM

**Comment by Andy-Python-Programmer** [comment]
Thanks @jwpjrdev!

**Pull request merged**